catalyst-gateway/bin/src/service/api/documents/mod.rs

```diff
@@ -103,10 +108,9 @@ impl DocumentApi {
     async fn post_document(
         &self, /// The Query Filter Specification
         query: Json<DocumentIndexQueryFilterBody>,
-        page: Query<Option<common::types::generic::query::pagination::Page>>,
-        limit: Query<Option<common::types::generic::query::pagination::Limit>>,
+        page: Query<Option<Page>>, limit: Query<Option<Limit>>,
         /// Authorization required.
-        _auth: CatalystRBACSecurityScheme,
+        _auth: NoneOrRBAC,
```

Comment:
Endpoint requires Authorization.
Don't change any security posture of any endpoint without signoff from architecture, ever.
Or is the intention to deliberately open the endpoint to a DDOS attack?